### ISSUE TITLE
"Property value expected type of" error for an unnamed function

### ISSUE THREAD

**Issue by yakirsobelman** [storybooks/babel-plugin-react-docgen#57]
Hi,

I not sure if this one relates to here or [babel-types](https://github.com/babel/babel/blob/master/packages/babel-types/README.md) but I hope someone here will be able to direct me to the right address.

I'm using `@storybook/react@4.0.0-alpha.14`.
When running `npm run storybook` I'm tackled with the error below (real filenames obfuscated):
```
ERROR in ./app/path/to/components/Component.js
Module build failed (from ./node_modules/babel-loader/lib/index.js):
TypeError: c:/dev/myapp/app/path/to/components/Component.js: Property value expected type of string but got null
    at Object.validate (c:\dev\app\node_modules\babel-types\lib\definitions\index.js:161:13)
    at validate (c:\dev\myapp\node_modules\babel-types\lib\index.js:505:9)
    at Object.builder (c:\dev\myapp\node_modules\babel-types\lib\index.js:466:7)
    at injectDocgenGlobal (c:\dev\myapp\node_modules\babel-plugin-react-docgen\lib\index.js:93:354)
    at c:\dev\myapp\node_modules\babel-plugin-react-docgen\lib\index.js:80:5
    at Array.forEach (<anonymous>)
    at injectReactDocgenInfo (c:\dev\myapp\node_modules\babel-plugin-react-docgen\lib\index.js:74:17)
    at PluginPass.exit (c:\dev\myapp\node_modules\babel-plugin-react-docgen\lib\index.js:14:11)
    at newFn (c:\dev\myapp\node_modules\babel-traverse\lib\visitors.js:276:21)
    at NodePath._call (c:\dev\myapp\node_modules\babel-traverse\lib\path\context.js:76:18)
 @ ./app/path/to/components/AnotherComponent.js 17:0-88 225:10-27
 @ ./stories/somesection.stories.js
 @ ./stories/index.stories.js
 @ ./.storybook/config.js
 @ multi ./node_modules/@storybook/core/dist/server/config/polyfills.js ./node_modules/@storybook/core/dist/server/config/globals.js ./.storybook/config.js (webpack)-hot-middleware/client.js?reload=true
```
After some research it appears that `Component.js` directly exports an unnamed function like this:
```javascript
export default (data) => (
  data.map(node => <div>{node.content}</div>)
)
```
When changing it to
```javascript
const func = (data) => (
  data.map(node => <div>{node.content}</div>)
)

export default func
```
Everything's good.
But it still looks like an issue cause there's nothing wrong with the first snippet as far as I understand.

**Comment by RyosukeCla** [comment]
same issue

**Comment by tiansijie** [comment]
i think is related to the issue here https://github.com/reactjs/react-docgen/pull/277

**Comment by danielduan** [comment]
published fix as `babel-plugin-react-docgen@2.0.0-rc.2`
